documentation/src/main/asciidoc/user_guide/query.adoc

```diff
@@ -814,9 +1447,302 @@ each query captures the minimal information needed both in terms of number of ma
 to quickly process all received events without blocking and to avoid performing actions that will lead to the generation
 of new matching events from the cache it listens to.
 
-=== More Query DSL samples
 
-Probably the best way to explore using the Query DSL API is to have a look at our tests suite.
-link:https://github.com/infinispan/infinispan/blob/master/query/src/test/java/org/infinispan/query/dsl/embedded/QueryDslConditionsTest.java[QueryDslConditionsTest]
-is a fine example.
+=== Remote Querying [[query.remote]]
+
+While previous Infinispan versions were already providing <<querying_infinispan,indexing and searching>> of Java
+entities to embedded clients, starting with Infinispan 6.0 and the introduction of the new
```

Comment:
(I'm not sure as it depends which aspect you mean.. if your really mean the 6.0 "new features" maybe there's no need to explain history anymore?)